docs/user/build-customization.rst

```diff
@@ -369,6 +369,15 @@ Supported :ref:`formats <downloadable-documentation:accessing offline formats>`
 * ``_readthedocs/pdf/``
 * ``_readthedocs/epub/``
 
+.. note::
+
+   Remember to create the folders before adding content to them.
+   You can ensure that the output folder exists by adding the following command:
+
+   .. code-block:: console
+
+       mkdir -p _readthedocs/html/
```

Comment:
Ideally, the whole page should use `$READTHEDOCS_OUTPUT/` instead of the hardcoded `_readthedocs/`. The environment variable is the absolute path and it will always create the correct directory.

I'm saying the whole page, because it's not just this note, but where the folder are listed in the previous bullet list, and the first paragraph as well.

Reply:
I was looking for an issue or PR about this yesterday but didn't find anything and forgot about it. Might as well do it here then :)

Reply:
We decided not to have env vars like `$READTHEDOCS_OUTPUT_HTML`, right?

Reply:
Updated examples in docs :+1: